wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -248,18 +259,15 @@ class PlatformRepo(val walletApplication: WalletApplication) {
             val profile = blockchainIdentity.watchProfile(10, 5000, BlockchainIdentity.RetryDelayType.SLOW20)
                     ?: throw TimeoutException("the profile was not found to be created in the allotted amount of time")
 
-            if (profile != null) {
```

Comment:
Good call on removing this null check.  This was copied from the recoveryIdentity method, but given that the watchProfile will return the profile document or throw an exception, we don't need to check for null.